index.html

```diff
@@ -28,6 +28,87 @@ <h1 class=" box-item headline"> Hey There.I'm Ahmed <br> <span id="headline-gree
+                    </ul>
+                </div>
+                <div class="flex-item4"><button>See Project</button></div>
+            </div>
+        </div>
+        <div class="main-container">
+            <div class="grid-item"> 
+                <div class="flex-item1"><img src="img/Plaholder.png" alt="placeholder"></div>
+                <div class="flex-item2"><h2>Multi-Post Stories Gain+Glory</h2></div>
+                <div class="flex-item3">
+                    <ul class="inner-flex-item">
+                        <li class="pill-btn">Ruby on rails</li>
+                        <li class="pill-btn">css</li>
+                        <li class="pill-btn">javaScript</li>
+                        <li class="pill-btn">html</li>
+                    </ul>
+                </div>
+                <div class="flex-item4"><button>See Project</button></div>
+            </div>
+        </div>
+        <div class="main-container">
+            <div class="grid-item"> 
+                <div class="flex-item1"><img src="img/Plaholder.png" alt="placeholder"></div>
+                <div class="flex-item2"><h2>Multi-Post Stories Gain+Glory</h2></div>
+                <div class="flex-item3">
+                    <ul class="inner-flex-item">
+                        <li class="pill-btn">Ruby on rails</li>
+                        <li class="pill-btn">css</li>
+                        <li class="pill-btn">javaScript</li>
+                        <li class="pill-btn">html</li>
+                    </ul>
+                </div>
+                <div class="flex-item4"><button>See Project</button></div>
+            </div>
+        </div>
+        <div class="main-container">
+            <div class="grid-item"> 
+                <div class="flex-item1"><img src="img/Plaholder.png" alt="placeholder"></div>
+                <div class="flex-item2"><h2>Multi-Post Stories Gain+Glory</h2></div>
+                <div class="flex-item3">
+                    <ul class="inner-flex-item">
+                        <li class="pill-btn">Ruby on rails</li>
+                        <li class="pill-btn">css</li>
+                        <li class="pill-btn">javaScript</li>
+                        <li class="pill-btn">html</li>
+                    </ul>
+                </div>
+                <div class="flex-item4"><button>See Project</button></div>
+            </div>
+        </div>
+        
+    </section>
+    <div class="bg-wrap">
+        <section class="aboutme-container">
+            <div class="wrap">
+                <h1 class=" box-item headline"> About Me</h1>
+                <p class="box-item">  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p> 
+                <div class="box-item fix"><button>Get My Resume</button></div>
+    
+            </div>
```

Comment:
- [ ] Kindly note that the about me section should be wrapped in a `section` tag and not a div💯

Reply:
Done